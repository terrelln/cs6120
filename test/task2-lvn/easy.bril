@main(a: int): int {
    b: int = const 1;
    c: int = const 1;
    d: int = const 1;
    print b;
    print c;
    ret d;
}