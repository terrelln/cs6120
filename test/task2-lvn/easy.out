@main(a: int): int {
._block0:
  b: int = const 1;
  print b;
  print b;
  print b;
  ret b;
}
